### PR TITLE
Modal when winning / losing

### PR DESCRIPTION
**Pull request by falt**

### CODE REVIEW

File: html/script/map2.js [hunk 1]
```diff
@@ -347,10 +353,18 @@ function disarmBomb () {
   let clue5 = document.forms['clueForm']['clue5'].value
   if (clue1 === answer[0] && clue2 === answer[1] && clue3 === answer[2] && clue4 === answer[3] && clue5 === answer[4] && timer > 0) {
     endTime = timer
-    alert('You made it! ***WINNING!*** ' + endTime)
+    //Winning modal
+    var centerControlDiv = document.createElement('div');
```
Comment (Alexosk) [inline html/script/map2.js]: Make VAR to LET and change name of variables

File: html/script/map2.js [hunk 2]
```diff
@@ -347,10 +353,18 @@ function disarmBomb () {
...
+    var centerControl = new WinningModal(centerControlDiv, gameMap);
```
Comment (Alexosk) [inline html/script/map2.js]: Make VAR to LET and change name of variables

**Review by mangster80 — APPROVED** [review]

File: html/script/map2.js [hunk 3]
```diff
@@ -347,10 +353,18 @@ function disarmBomb () {
...
+    gameMap.controls[google.maps.ControlPosition.CENTER].push(centerControlDiv);
+
     document.getElementsByClassName('timer')[0].style.display = 'none'
   } else {
-    alert('You are dead AF!')
+    // Losing modal 
+    var centerControlDiv = document.createElement('div');
```
Comment (Alexosk) [inline html/script/map2.js]: Make VAR to LET and change name of variables

File: html/script/map2.js [hunk 4]
```diff
@@ -347,10 +353,18 @@ function disarmBomb () {
...
+    var centerControl = new LosingModal(centerControlDiv, gameMap);
```
Comment (Alexosk) [inline html/script/map2.js]: Make VAR to LET and change name of variables

File: html/script/modal.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
+
+function WinningModal(controlDiv, gameMap) {
```
Comment (Alexosk) [inline html/script/modal.js]: Fix VAR to LET and change names

**Review by Alexosk — CHANGES_REQUESTED** [review]

File: html/script/map2.js [hunk 6]
```diff
@@ -321,7 +322,11 @@ function countDownTimer (duration, display) {
     display.textContent = minutes + ':' + seconds
 
     if (--timer < 0) {
-      alert('You have lost the game')
+      // Losing modal 
+      var centerControlDiv = document.createElement('div');
```
Comment (mirandamutka) [inline html/script/map2.js]: var to let and remove ;

**Review by mirandamutka — CHANGES_REQUESTED** [review]
change var to let and remove ;

**Review by mirandamutka — APPROVED** [review]

**Review by Alexosk — APPROVED** [review]

**Pull request merged**